### PR TITLE
Use tumbleweed as a base image

### PR DESCRIPTION
**Pull request by andrewscat**
environment variable SYSTEMCTL_FORCE_BUS makes systemctl work inside the conatiner

### CODE REVIEW

**Comment by andrewscat** [comment]
@inercia @ereslibre can you please take a look

File: Dockerfile [hunk 1]
```diff
@@ -1,33 +1,50 @@
...
-# ARG EXTRA_REPO0="https://download.opensuse.org/repositories/devel:/kubic/openSUSE_Tumbleweed/"
+ARG EXTRA_REPO0="https://download.opensuse.org/repositories/devel:/kubic/openSUSE_Tumbleweed/"
 
 # for Leap
-ARG EXTRA_REPO0="https://download.opensuse.org/repositories/devel:/CaaSP:/Head:/ControllerNode/openSUSE_Leap_15.0/"
+# ARG EXTRA_REPO0="https://download.opensuse.org/repositories/devel:/CaaSP:/Head:/ControllerNode/openSUSE_Leap_15.0/"
 
-ARG RUN_RPMS="cri-tools iptables iproute2 systemd"
+# This is needed to make systemctl work inside the container
+ENV SYSTEMCTL_FORCE_BUS="1"
```
Comment (ereslibre) [inline Dockerfile]: :clap:

File: Dockerfile [hunk 2]
```diff
@@ -1,33 +1,50 @@
-FROM opensuse:leap
+FROM opensuse:tumbleweed as builder
```
Comment (ereslibre) [inline Dockerfile]: I don't see a big benefit in the multistage build, as it's a bit slower compared to the one we had; the idea is to eventually stop building with this Dockerfile and produce this container with OBS directly. Right now, it's just a way to build it while we lay out the final way.

Before this change (that makes us un-ignore more folders on `.dockerignore`) our context sent to the docker daemon was around 70Mb, now it's more with the slowdown it brings.

Comment (andrewscat) [reply Dockerfile]: Oh, I see. That makes sense.
I noticed a TODO and decided to go the extra mile :)
At the same time a small misunderstanding took place.

**Comment by ereslibre** [comment]
@andrewscat We can merge as is, but could you squash the commits, please?

**Pull request merged**